oss-internship-2020/openjpeg/examples/decompress_example.cc

```diff
@@ -0,0 +1,159 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Perform decompression from *.jp2 to *.pnm format
+
+#include <libgen.h>
+#include <syscall.h>
+
+#include <cassert>
+#include <cstdlib>
+#include <iostream>
+
+#include "convert_helper.h"
+#include "openjp2_sapi.sapi.h"
+#include "sandboxed_api/util/flag.h"
+
+class Parameters : public sapi::v::Struct<opj_dparameters_t> {};
```

Comment:
those are used just once in the code below and I don't think it improves redability - e.g it's not immediately visible that `Opj_image_t` is a `sapi::v::Struct`